contracts/child/IParentToken.sol

```diff
@@ -1,7 +1,7 @@
-pragma solidity ^0.4.24;
+pragma solidity ^0.5.2;
 //interface for parent contract of any child token
 
 
-interface IParentToken {
+contract IParentToken {
```

Comment:
How about we let this be an interface and make `beforeTransfer` external.

Reply:
beforeTransfer won't be called externally 😆.

Reply:
can't, `beforeTransfer ` will never be called externally.

Reply:
It will be, but let's ignore it for now :)

Reply:
reverted back to interface.